### ISSUE TITLE
Art: Lisk Witch character ('Ann')

### ISSUE THREAD

**Issue by avaer** [webaverse/app#1386]
Description: Extremely weird and shy girl, some people think she might be mute but that’s not true. Doesn’t think much about her appearance and wears the same thing for years. She mostly has a single iconic design of some sort of old fashioned torn robe, with an ethereal fantasy nature to it.
Color: Green

# Face
![img](https://cdn.discordapp.com/attachments/842248635948597279/867430396379922482/unknown.png)
![image](https://user-images.githubusercontent.com/6926057/126523127-bd980090-f3ae-431d-9ef6-ecec2eb96aba.png)

# Body/clothes
![img](https://cdn.discordapp.com/attachments/842248635948597279/867430438863372317/unknown.png)

The level of tattering on the robes doesn't matter, but we should make it flowing and flag it as VRM "hair". That might also need colliders.

# Weapon
The scissors are important here because they are supposed to go on the back like wings.

![scissors](https://user-images.githubusercontent.com/6926057/126524883-d665d2cd-34ac-4904-ad34-c819b46d7dd0.png)
![img](https://camo.githubusercontent.com/0cd0eb8c0b1bb9febb52c33447c9f2273ea3a2e49f1b1b871f6160bc1f1c4f07/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f6174746163686d656e74732f3832333239393439313039313834313036352f3836333230393138393535303538373934342f696d616765302e706e67)

**Comment by avaer** [comment]
I saw this and thought it was a good alt/work clothing (if we go with duals). I don't think we should make this first but it might help factor some things in:

![img](https://i.imgur.com/iBCHIHJ.jpg)

**Comment by GaladWarder** [comment]
Finally figured out what this style of dress/clothing is called - Mori Kei.

Was trying to figure that out for longer than I'd care to admit while looking for refs, the keyword may be useful later.

**Comment by avaer** [comment]
Awesome, thanks for finding that! I had no idea, I was just going off of a Pin.

**Comment by GaladWarder** [comment]
Yeah, it's such a freaking cool aesthetic, I was hyped when I found it.

Alright, uploading some work here; this is the bash I came up with for an initial shot at the witch concept. Should at least give a starting point to work from!

![Witch_V1](https://user-images.githubusercontent.com/29031531/126806223-04f79725-a473-4793-9580-29143b52533c.jpg)

**Comment by Vianvolaeus** [comment]
Just want to confirm we're set on the above concept. I'm confident #1382 is nearing completion and I can start running this in parallel while changes need to be reviewed in that issue.

**Comment by GaladWarder** [comment]
+1 on the greenlight request, I can do the lines and color flats for the posed character shot, as well as ortho's once it's locked in

**Comment by avaer** [comment]
Can we de-puff the outer jacket/get rid of it?

I think this is supposed to be a bit of a stealth character, and although she is fine to be fluffy and covered up, the jacket feels like it would be bad for stealth.

She should be somewhat graceful and flowing, rather than clogging the hallway :P.

The ribbons are a great idea but IMO should flow behind rather than getting in the way at the front.

The rest seems correct so far.

**Comment by GaladWarder** [comment]
Can definitely de-puff or replace the outer jacket, should be easy. Same with the ribbons, I'll just move them both to behind. Will have a V2 posted soon

**Comment by GaladWarder** [comment]
Here's a second pass at the Witch!

- Slimmed up the overall silhouette, made it more 'arrowlike' so she will look stealthy/sleeker
- Kept the ribbons, but moved to back and adjusted silhouettes of them
- changed the jacket completely, went for something that feels more graceful and flowing, has a bit of JP influence too.

![Witch_V2](https://user-images.githubusercontent.com/29031531/128447953-c2830556-b244-4318-b226-82807d392236.jpg)

**Comment by avaer** [comment]
I think it works, we can start to model. 👍 

The book interesting, very anachronistic. If someone would have a book it would be her. Frankly I think all of the wearables make sense.

**Comment by avaer** [comment]
I like the modern arrowlike part here too:

![image](https://user-images.githubusercontent.com/6926057/128495781-b8b8aee7-b65d-4f53-8780-f378d7e1a63a.png)

It's overall a great interpretation. 👏

**Comment by GaladWarder** [comment]
Stylesheet for Ann, keeping consistent with other character sheets:
![Witch_V3](https://user-images.githubusercontent.com/29031531/129099996-5bf06666-a804-443b-ab1b-e72bdf7c873b.jpg)

Modelling orthos are coming asap.

Also, we will probably need/want to make a sheet for the design of the book and bag that she is wearing; didn't do it right now because I'm not sure if the props are relevant enough to be high-priority work. Can circle back around on them and make model sheets when needed, though.

Didn't bother with the scissors since they are already done; just let me know if you want them included in this image and I can add them in pretty quickly.

**Comment by avaer** [comment]
Looks good to model.

I think the books could just be wearable like Scillia’s hat, but they do look good with the character.

**Comment by madjin** [comment]
Ann pureref: https://cdn.discordapp.com/attachments/641036402829819914/875787805014515722/ann.pur
![ann2](https://user-images.githubusercontent.com/32600939/129395389-7801a7f8-bd4a-42f1-bcda-68406680261e.jpg)

**Comment by GaladWarder** [comment]
Orthos done:

![Witch_Orthos_V2](https://user-images.githubusercontent.com/29031531/129422016-4b72219e-8296-4d70-b263-49d61a5936a3.jpg)

I have started the book and pack, but since they will be modelled separately as accessories anyway, I didn't include them in this sheet to keep things clean/readable. I also get the vibe that they are lower priority, so I'll move on to Hya first and then loop back as needed.

The foot armor is fairly high-density detail, depending on the art direction it could be simplified if needed.

**Comment by Vianvolaeus** [comment]
I'll be moving onto this one next if we're happy with the above, since #1390 appears to be unconfirmed / not ready for modelling yet. 

> The foot armor is fairly high-density detail, depending on the art direction it could be simplified if needed.

The base meshes themselves will be low detail with material/tex work doing the heavy lifting on any potential details 👍

**Comment by GaladWarder** [comment]
Do you need/want the orthos for the book and pack, or do you want to just design as you go? I have orthos started, but got shifted to other projects in the meantime.

If you do end up needing them, I think it'd just take an hour or so to finish them off, so I could fit them in this week or at the beginning of next; whatever's easiest for you.

**Comment by Vianvolaeus** [comment]
No rush on them immediately, don't think they'll hold anything up - can push to next week

**Comment by Vianvolaeus** [comment]
Hair's temp for now - started high res sculpt on top of this for detail as well, but it's blocked out and fit
![image](https://user-images.githubusercontent.com/80061436/131878437-2734a17f-02ce-49e1-8865-6772c25fbf50.png)

**Comment by Vianvolaeus** [comment]
Didn't post progress - wrestled with this outfit- high poly sculpt for details, UVs done, mat work first pass more or less done: 
![image](https://user-images.githubusercontent.com/80061436/132066011-648b8ef3-0950-40d1-9bc2-e0dfee1eea13.png)

**Comment by Vianvolaeus** [comment]
Head is pretty much placeholder, but the hair looks pretty on similar to what we're looking for

![image](https://user-images.githubusercontent.com/80061436/132066618-92faa7f1-cbc2-4147-94f7-4b28b0f77067.png)

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/132066717-987293cc-17fd-4f9a-87fc-21f7f220574e.png)
Color only orthos

**Comment by Vianvolaeus** [comment]
With scissor weapon:
![image](https://user-images.githubusercontent.com/80061436/132067065-978807ef-5155-4e95-98ec-fe679f1aa5b0.png)

**Comment by avaer** [comment]
I love the eyes, it's looking really good!

Needs a color pass/some tweaks the clothes and hair color. Let's do with dark/blue hue on the hair. Red is dominant but it's a taken color (June) so we should make the theme green. The robes I interpret as more silky/victorian, rather than celtic/warm in winter.

![](https://images.squarespace-cdn.com/content/v1/5a0875c9c027d8e184a1cbee/1584368419975-VPS2IQFG8CROOQRK3P8E/gray+victorian+dress.jpg?format=1000w)
![](https://miro.medium.com/max/1400/0*Tf9B4IGX1Zqebwbv.jpg)
![](https://i.pinimg.com/736x/ef/8c/1c/ef8c1c63dfd7e2b099d2096102b12890.jpg)

Also, it's mostly an artifact of the colors, but the leathery texture of the hands makes it look a little bit unnerving to me, like very huge/infected hands :P

![image](https://user-images.githubusercontent.com/6926057/132087770-9ce9a8b0-7440-4e88-ad92-8fad619aff8b.png)

It might be better to just have the sleeves be uniform and instead put details into the silk pattern.

**Comment by avaer** [comment]
By the way, I couldn't find this model in the GDrive. I assumed it would have been in New Concepts.

**Comment by Vianvolaeus** [comment]
It's not fully rigged / ready to pose yet, didn't want to cause confusion by uploading it unfinished.

**Comment by Vianvolaeus** [comment]
Made a quick rigging pass (will need to be properly done, these kinds of outfits take a bit of work to get them done cleanly), as well as a quick file cleanup. It should be enough to pose in a scene, but it's not game-ready.

**Comment by Vianvolaeus** [comment]
Redone material pass: 

![image](https://user-images.githubusercontent.com/80061436/132353181-63085043-c257-4eb8-a325-fda6e2fdd463.png)
![image](https://user-images.githubusercontent.com/80061436/132353206-05b4c1bd-e77a-4fed-9aa0-3db640887f61.png)

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/132354156-0a5f2e4a-8e44-415c-b6fc-59aa35ce6db4.png)

**Comment by Vianvolaeus** [comment]
Also a much more green leaning one
![image](https://user-images.githubusercontent.com/80061436/132354872-d8b2a89d-0dd9-4028-a0fc-1bf387f4f3d5.png)

**Comment by Vianvolaeus** [comment]
@GaladWarder would love if you could post your thoughts regarding color etc here since i'm pretty sieve-brained at the moment. 

I'll take another pass at colors / roughness tomorrow!

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/132522358-198c06e9-8e21-4893-b760-24ae9b45a533.png)

More color passes based on the idea of desaturating the main area of the outfit. Tried a different material pattern as well.

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/132540407-1ed07490-3777-409e-abd0-7d7ea24e3b0a.png)

Styled up the hair a little to match the concept a bit more, but make it a little less uniform. Rigged, very few bones in it as well.

**Comment by avaer** [comment]
Sick work!

The left ones look the cleanest to me. I like the bottom left the best, I just don't know if when posed it would look like massive hands instead of sleeves. Probably not.

![image](https://user-images.githubusercontent.com/6926057/132541466-1eea5f76-adc9-4554-9a1a-1841d1824661.png)

It's probably also worth changing the hue of the scarf (especially the tail) to make it more distinct from the skirt.

The hair is very nice.

Re: the eyes, would it read cleaner with some sort of line or pattern work? The color is nice but it feels just slightly flat. Don't read too much into these, I'm sure something else would look better:

![image](https://user-images.githubusercontent.com/6926057/132541240-5a8256c7-0d3a-4ea4-87ca-444752ec2409.png)

**Comment by avaer** [comment]
For scarf, blue/blackish or even more violet scarf would work. IMO this is pretty good, but could be more vibrant with a more violet, which tends to go well with greens.

![image](https://user-images.githubusercontent.com/6926057/132542026-8a74669e-8aca-4598-bc2f-3ef544837573.png)

**Comment by Vianvolaeus** [comment]
Gonna do a push on face / body tex tomorrow, these were just temporary 👍 It'll be a big difference on the eyes especially

**Comment by Vianvolaeus** [comment]
Updated colors! Left is diffuse only, right is with studio hdri

![image](https://user-images.githubusercontent.com/80061436/132719631-63893d9b-079d-4d3f-93e7-c591eb2e3553.png)

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/132719727-0e817138-cb21-47a2-8264-1d4d4f5e712b.png)
Updated the skin textures and done eyes. Hair texture is a little different as it was a bit flat before, just darkened the roots up slightly.

**Comment by avaer** [comment]
Love the updated skin textures and eyes! We'll like have to do more debugging for VRM, but otherwise this looks done to me 💯.

**Comment by avaer** [comment]
Heads up, for this one I think it might draw better without the belt. There's nothing wrong with it, especially in 3d, but I'm not sure its presence strengthens the shape language when converted to 2d; it might just be an extra thing that is hard to draw from different angles.

![image](https://user-images.githubusercontent.com/6926057/134089887-518ce242-1dc6-4634-ac9a-5a353c152ab2.png)
vs
![image](https://user-images.githubusercontent.com/6926057/134089858-5e0def52-3de9-4391-a300-d981f5d4b12e.png)

**Comment by Vianvolaeus** [comment]
Removed the belt and fixed up textures. This was unexpectedly a PITA to rebake and fix.

![image](https://user-images.githubusercontent.com/80061436/134373598-03f82ce1-6bce-4b29-81f8-f0c01e2a4ea0.png)

**Comment by avaer** [comment]
This looks amazing, the mats are great!

If the bake procedure is down, can we try with the belt enabled? I think my hunch was wrong, and in 2D art tests the belt turned out to work well, with a slightly tweaked design (thin + different amulet).

![image](https://user-images.githubusercontent.com/6926057/134598345-712482bd-2aa9-4da3-aeab-b534e3933970.png)
![](https://cdn.discordapp.com/attachments/806810240237568052/890371339821658143/image2.jpg)

**Comment by Vianvolaeus** [comment]
Should be able to fit it in without issue.  I'll keep it separate from the rest of the outfit to avoid any problems. I'll assume that the cracked / altas-ish design is what we're going for in terms of the buckle?

**Comment by avaer** [comment]
I think it's supposed to be weird lisk fangs, but the zigzags tie to the bitemark. Just vaguely that shape.

**Comment by Vianvolaeus** [comment]
![image](https://user-images.githubusercontent.com/80061436/134720070-93c9b84c-8a2b-41a4-8cf5-00085c2146ad.png)
Updated on drive, added in new belt.

**Comment by avaer** [comment]
The modeling on that medallion is really pretty, like a perfect match to the art. 🤯

I thought the belt could be a thin plane mesh but this looks great.

I don’t think there is anything left on the design side, just the technical work of visemes/rigging/vrm.

**Comment by NotFuji** [comment]
![ann_v2](https://user-images.githubusercontent.com/38316883/135899354-71e9e0d3-f199-49b6-870a-599f3f4487e6.png)

**Comment by Vianvolaeus** [comment]
v1 VRM is in the Drive! 

https://drive.google.com/drive/folders/1mP5Y3mqxJrpka8tiy_yyl6sf-_0qQDrf
